2.evaluate_models/log_reg_plate_2_cp_fs_data/nbconverted/evaluate_model.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
```

Comment:
Similar comment about some redundancy.

Reply:
Agreed, I will apply the suggestion mentioned above.